packages/api/src/tasks/checkAlerts.ts

```diff
@@ -58,25 +56,110 @@ export const buildLogSearchLink = ({
   return url.toString();
 };
 
-const buildEventSlackMessage = ({
+// TODO: should link to the chart instead
+export const buildChartLink = ({
+  dashboardId,
+  endTime,
+  granularity,
+  startTime,
+}: {
+  dashboardId: string;
+  endTime: Date;
+  granularity: string;
+  startTime: Date;
+}) => {
+  const url = new URL(`${config.FRONTEND_URL}/dashboards/${dashboardId}`);
+  const queryParams = new URLSearchParams({
+    from: startTime.getTime().toString(),
```

Comment:
doesn't this build a pretty terrible dashboard where you only get 1 data point per graph? I think in the URL we should have a time range that's probably like +/- 7*granularity so you get at least 14 data points instead of just 1

Reply:
sounds good 👍